limbus/core/component.py

```diff
@@ -364,3 +369,49 @@ def init_pipeline(self) -> None:
 
         """
         pass
+
+
+try:
+    class TorchComponent(BaseComponent, nn.Module):
```

Comment:
this class should go in `limbus-components` library

Reply:
now it does not exist